ethdb/database.go

```diff
@@ -189,6 +197,7 @@ type ResettableAncientStore interface {
 type Database interface {
 	Reader
 	Writer
+	RangeDeleter
```

Comment:
Please extend the Writer with `RangeDeleter` function

Reply:
done